src/components/attendee/NavOverlay/ProfileOverlay/AudioSettings/AudioSettings.tsx

```diff
@@ -0,0 +1,22 @@
+import { Select } from "components/attendee/Select";
+
+import { FormFieldProps } from "types/forms";
+
+type AudioSettingsProps = {
+  register: FormFieldProps["register"];
```

Comment:
This is an old definition. Shouldn't it be `UseFormRegister` type ?

Reply:
done [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)